ranchhand.tf

```diff
@@ -1,28 +1,36 @@
 locals {
-  ranchhand_cert_ips = ["${concat(list(local.lb_ip), var.ranchhand_cert_ipaddresses)}"]
-  public_ips         = "${split(",", var.enable_public_instances ? join(",",azurerm_public_ip.vm.*.ip_address) : join(",",azurerm_network_interface.vm.*.private_ip_address))}"
+  ranchhand_cert_ips = concat([local.lb_ip], var.ranchhand_cert_ipaddresses)
+  public_ips = split(
+    ",",
+    var.enable_public_instances ? join(",", azurerm_public_ip.vm.*.ip_address) : join(",", azurerm_network_interface.vm.*.private_ip_address),
+  )
 
-  node_ips = "${var.enable_public_instances ?
-    join(",", formatlist("%v:%v", local.public_ips, azurerm_network_interface.vm.*.private_ip_address)) :
-    join(",", azurerm_network_interface.vm.*.private_ip_address)}"
+  node_ips = var.enable_public_instances ? join(
+    ",",
+    formatlist(
+      "%v:%v",
+      local.public_ips,
+      azurerm_network_interface.vm.*.private_ip_address,
+    ),
+  ) : join(",", azurerm_network_interface.vm.*.private_ip_address)
```

Comment:
Can we use this pattern? 

https://github.com/cerebrotech/terraform-aws-rancher/blob/8d559f432bc09bfbb8e7bb323262470290670c62/main.tf#L353-L357

Reply:
Yep!